### DISCUSSION TITLE
toggles fullscreen on spacebar press, insted of toggling play/pause

### DISCUSSION THREAD

**Discussion by byungjuJin** [vimeo/player.js#982] [Bug reports]
### Expected Behavior
when click "Spacebar" on keyboard, start or stop Playing.

### Actual Behavior
when click "Spacebar" on keyboard, takes to fullscreen or vice versa

### Steps to Reproduce
1. Make a fullscreen with button
2. Make small screen with button
3. click "Spacebar" on keyboard
(It works like focus on fullscreen-button, and spacebar clicking makes it full or small.)

Behavior is nealy same with below ::

https://stackoverflow.com/questions/64063002/why-videojs-toggles-fullscreen-on-spacebar-press-insted-of-toggling-play-pause

you can check on demo page as well ::
https://player.vimeo.com/api/demo

Nowhere shows keyboard shotcut for Spacebar ::
https://vimeo.zendesk.com/hc/en-us/articles/115002267667-Player-keyboard-shortcuts

**Comment by Badsheep** [comment]
Hello,

Same bug here with chrome on Mac OS.
But it works with Safari (the Spacebar touch toggles play and pause)

**Comment by gabrielKerekes** [comment]
I can also confirm the same behaviour on Mac OS with Brave and Firefox. Safari works as expected 😮

**Comment by jackmcdade** [comment]
This is driving me nuts – has anyone found a workaround for this bug?

**Comment by felipevaz** [comment]
I only noticed it after migrating to a Touch Bar macbook pro, so it might be related.

And I the thing is: when you enter fullscreen by clicking on its icon, that's the last thing selected. So since there's no shortcut active for the spacebar, it clicks/activates whatever you last clicked. If you click in the middle of the screen to pause once, then the spacebar will work as pause/play.
